utils/onchain/evm.ts

```diff
@@ -0,0 +1,51 @@
+import { http, createConfig, readContract } from '@wagmi/core'
+import { base, immutableZkEvm } from '@wagmi/core/chains'
+import type { Prefix } from '@kodadot1/static'
+import { GENSOL_ABI } from '@/composables/transaction/evm/utils'
+import { getEvmCollection } from '@/services/ogi'
+
+export const wagmiConfig = createConfig({
+  chains: [base, immutableZkEvm],
+  transports: {
+    [base.id]: http(),
+    [immutableZkEvm.id]: http(),
+  },
+})
+
```

Comment:
can we use the same wagmi config? useWagmi()

Reply:
any idea why I can't use config from useWagmi()? it throw error on readContract

Reply:
Oh, I just saw this PR: https://github.com/kodadot/nft-gallery/pull/10762/files
. It will probably work once that one is merged. Since there is wagmi config in the plugin